README.md

```diff
@@ -18,6 +18,63 @@ To install [the current release](https://pypi.org/project/nvflare), you can simp
 pip install nvflare
 ```
 
+## Quick Start
+
+ * [Getting Started](https://nvflare.readthedocs.io/en/latest/getting_started.html)
+ * [Examples](https://github.com/NVIDIA/NVFlare/tree/2.1/examples/)
```

Comment:
The branch should NOT be 2.1
If we are talking about latest -> should change to dev
If we will create a 2.2 branch or something -> should change to that branch

Reply:
If after release things will be on main branch -> should change to "main"